lib/kadira.js

```diff
@@ -334,6 +336,11 @@ Kadira.startContinuousProfiling = function () {
   });
 };
 
+/**
+ * @warning Mutating the `__meteor_runtime_config__` object does not propagate in real-time to the client, only if the
```

Comment:
@zodern Do you think we should backlog this?

Reply:
At this point I don't think updating in realtime is necessary.